msg/vehicle_local_position.msg

```diff
@@ -52,6 +52,10 @@ float32 ref_alt				# Reference altitude AMSL, (metres)
 # Distance to surface
 float32 dist_bottom			# Distance from from bottom surface to ground, (metres)
 bool dist_bottom_valid			# true if distance to bottom surface is valid
+uint8 dist_bottom_sensor_bitfield	# bitfield indicating what type of sensor is used to estimate dist_bottom
```

Comment:
Do you expect certain consumers will be checking this?

Reply:
Yes, the land detector; we'll push some updates soonish